README.md

```diff
@@ -16,23 +35,71 @@ are not real.
 * **Release Stage:** Alpha
 * **Self-service / Independent Setup:** Follow the instructions to set up Emblem by reading the [setup quickstart](./docs/tutorials/setup-quickstart.md), or by launching the [Interactive Walkthrough](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Femblem&cloudshell_tutorial=docs%2Ftutorials%2Fsetup-walkthrough.md) on Cloud Shell.
 
-## Learn About This Project
-
-* This project will help you learn about:
-  * 2-tier web application architecture with Cloud Run
-  * Continuous Delivery with Cloud Build
-  * Using Terraform to manage Google Cloud resources
-  * Making sustainable decisions on Cloud usage for a team
-* See products in use: Cloud Run, IAM, Secret Manager, Cloud Storage,
-  Cloud Firestore, Cloud Build, Artifact Registry, Cloud Pub/Sub, Cloud Logging
-* Go deeper into project details in the [documentation](./docs)
-* Read through the [technical decisions](docs/decisions/README.md) that got us where we are today.
-
 ## Contributing
 
 * Become a [CONTRIBUTOR](./CONTRIBUTING.md)!
 * Check out our shovel-ready [Good First Issues](https://github.com/GoogleCloudPlatform/emblem/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) or go a bit deeper in [Help Wanted](https://github.com/GoogleCloudPlatform/emblem/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22)
 
+## Getting Started
+
+Emblem is made of a combination of resources created and managed by Terraform and resources created via the Google Cloud CLI or Google Cloud Console. You may deploy Emblem by either running `setup.sh` (see [streamlined setup instructions](#streamlined-setup)) or by following the [manual steps below](#manual-setup). 
+
+### Prerequisites
+
+To deploy Emblem, you will need:
+  * 3 Google Cloud projects (ops, stage, prod) with billing enabled on each
+  * A fork of this repo
+
+Your local host will need the following installed:
+  * Google Cloud CLI
+  * Terraform
+  * Python3
+
+We recommend running through setup steps using Google Cloud Shell, which has the required softare pre-installed. The following will open Cloud Shell Editor and clone this repo:
+
+ [![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Femblem&cloudshell_tutorial=docs%2Ftutorials%2Fsetup-walkthrough.md)
+
+### Streamlined setup
+
+1. **Connect a fork of this Github repo to your Emblem ops project**
+   
+   Click [here](https://console.cloud.google.com/cloud-build/triggers/connect) to connect your repository to your Google Cloud Project. Make sure you are working from your **ops** project. This will require you to authenticate with your Github account.
```

Comment:
here should be a description rather than just "here" so folks know what they are clicking on.

Reply:
in other words I'm not clicking on here based on that sentence :)